entries/b/bookbeat.fi.json

```
@@ -0,0 +1,17 @@
+{
+  "BookBeat": {
+    "domain": "bookbeat.fi",
```

As BookBeat operates internationally, an international TLD is preferred over a local one.
```suggestion
    "domain": "bookbeat.com",
```